neondatabase/website
# chore: add neon logo to InKeep chat

This pull request introduces the Neon logo into the InKeep chat integration interface.

Steps to check:

- open [preview](https://neon-next-git-inkeep-chat-neon-logo-neondatabase.vercel.app/docs/introduction)
- make sure that everything looks and work as expected
